airflow/roles/airflow/tasks/main.yml

```diff
@@ -4,23 +4,67 @@
     name: airflow-stable
     repo_url: https://airflow-helm.github.io/charts
 
+# create in line sealed secret since ansible variables don't propagate  all the way to the k8s manifests
+- name: Create secret for database connection
+  community.kubernetes.k8s:
+    state: present
+    namespace: airflow
+    definition:
+      apiVersion: bitnami.com/v1alpha1
+      kind: SealedSecret
+      metadata:
+        creationTimestamp: null
```

Comment:
do you need to set this and below as null?

Reply:
True, we probably won't need it. Will make another PR some time to clean it up.